swift/apple/Firezone/xcconfig/debug.xcconfig

```diff
@@ -1,6 +1,6 @@
 // Apple Developer account-specific configuration
 DEVELOPMENT_TEAM = 47R2M6779T
 PRODUCT_BUNDLE_IDENTIFIER = dev.firezone.firezone
-APP_GROUP_ID[sdk=macosx*] = 47R2M6779T.group.dev.firezone.firezone
```

Comment:
This actually wasn't named correctly, which will cause issues when installing the SystemExtension because this identifier naming pattern is used as a convention to associate / enforce that the SystemExtension belongs to the containing app.

Reply:
For the sake of keeping the PR as focused as possible, can we split it out?

Reply:
This one we can't - without this change, the System Extension fails to load.